conftest.py

```diff
@@ -26,6 +26,11 @@
 except ImportError:
     collect_ignore_glob.append("dask/dataframe/*")
 
+try:
+    import fsspec  # noqa: F401
+except ImportError:
+    collect_ignore_glob.extend(["dask/bag/*", "dask/bytes/*"])
```

Comment:
I'm surprised to see us skipping all the Dask bag tests if `fsspec` isn't installed. There are tests in, for example, `dask/bag/tests/test_bag.py` which don't require any filesystem operations.

Reply:
Yeah, I agree but we import bytests in bag and bytes needs fsspec. We could separate it out, but in light of https://github.com/dask/dask/issues/7244 it didin't seem worth it.

Reply:
I see, thanks for clarifying. Yeah, agreed in light of #7244 this seems okay